eclair-core/src/test/scala/fr/acinq/eclair/wire/protocol/PaymentOnionSpec.scala

```diff
@@ -162,6 +164,30 @@ class PaymentOnionSpec extends AnyFunSuite {
     assert(encoded == bin)
   }
 
+  test("encode/decode node relay to blinded paths per-hop payload") {
+    val features = Features(Features.BasicMultiPartPayment -> FeatureSupport.Optional).toByteVector
+    val blindedRoute = OfferTypes.CompactBlindedPath(
+      OfferTypes.ShortChannelIdDir(isNode1 = false, RealShortChannelId(468)),
+      PublicKey(hex"0232882c4982576e00f0d6bd4998f5b3e92d47ecc8fbad5b6a5e7521819d891d9e"),
+      Seq(RouteBlinding.BlindedNode(PublicKey(hex"03823aa560d631e9d7b686be4a9227e577009afb5173023b458a6a6aff056ac980"), hex""))
+    )
+    val path = PaymentBlindedContactInfo(blindedRoute, OfferTypes.PaymentInfo(1000 msat, 678, CltvExpiryDelta(82), 300 msat, 4000000 msat, Features.empty))
+    val expected = TlvStream[OnionPaymentPayloadTlv](AmountToForward(341 msat), OutgoingCltv(CltvExpiry(826483)), TotalAmount(1678 msat), OutgoingBlindedPaths(Seq(path)), InvoiceFeatures(features))
+    val bin = hex"86 02020155 04030c9c73 1202068efe000 1023103020000fe000 102366a0100000000000001d40232882c4982576e00f0d6bd4998f5b3e92d47ecc8fbad5b6a5e7521819d891d9e0103823aa560d631e9d7b686be4a9227e577009afb5173023b458a6a6aff056ac9800000000003e8000002a60052000000000000012c00000000003d09000000"
```

Comment:
I believe your whitespaces are wrong and don't separate the TLV fields correctly, starting at `fe000`:

```suggestion
    val bin = hex"86 02020155 04030c9c73 1202068e fe0001023103020000 fe000102366a0100000000000001d40232882c4982576e00f0d6bd4998f5b3e92d47ecc8fbad5b6a5e7521819d891d9e0103823aa560d631e9d7b686be4a9227e577009afb5173023b458a6a6aff056ac9800000000003e8000002a60052000000000000012c00000000003d09000000"
```